test/functional/treesitter/inspect_tree_spec.lua

```diff
@@ -120,22 +120,24 @@ describe('vim.treesitter.inspect_tree', function()
   end)
 
   it('updates source and tree buffer windows and closes them correctly', function()
+    local name = t.tmpname()
+    n.command('e ' .. name)
```

Comment:
use full command names in code. helps when searching for usages. same for the rest of this file
```suggestion
    n.command('edit ' .. name)
```